packages/eslint-plugin/src/rules/ban-ts-comment.ts

```diff
@@ -98,20 +98,21 @@ export default util.createRule<[Options], MessageIds>({
     },
   ],
   create(context, [options]) {
-    const tsCommentRegExp = /^\/*\s*@ts-(expect-error|ignore|check|nocheck)(.*)/;
+    const commentDirectiveRegExSingleLine = /^\/*\s*@ts-(expect-error|ignore|check|nocheck)(.*)/;
+    const commentDirectiveRegExMultiLine = /^\s*(?:\/|\*)*\s*@ts-(expect-error|ignore|check|nocheck)(.*)/;
```

Comment:
It would be great to link back to the typescript source code as reference here, should these ever need to change (as they did recently when expect error was introduced)

Reply:
Added the reference to the typescript docs here.

Reply:
thanks